src/main/java/org/opentripplanner/middleware/otp/OtpRequestProcessor.java

```diff
@@ -139,6 +140,7 @@ private static boolean saveTripRequest(TripRequest tripRequest) {
         } catch (MongoException e) {
             success = false;
             LOG.error("Unable to save trip request: " + tripRequest, e);
+            BugsnagReporter.reportErrorToBugsnag(e, "Unable to save trip request");
```

Comment:
Is there a convenient way to log the message inside the `reportErrorToBugsnag` method (ideally without losing the line number/stack trace to come back to)? Just because it's a bit redundant to have the same message repeated. Also, it seems valuable to include tripRequest details here as well (or was there a reason for not doing that?).

Reply:
This is misleading. It is actually the context, so Bugsnag groups errors by this value. I'm going to refactor as additional information is required.

Reply:
Updated to be context, message, exception. The message is saved as app info which is basically a key/value pair. It is possible to save a lot more information under app info, but to keep the Bugsnag reporting call to a minimum only a single item can be provided.